website/source/docs/builders/openstack.html.md

```diff
@@ -166,6 +171,48 @@ builder.
 -   `security_groups` (array of strings) - A list of security groups by name to
     add to this instance.
 
+-   `source_image_filter` (object) - Filters used to populate filter options.
+    Example:
+
+    ``` json
+    {
+        "source_image_filter": {
+            "filters": {
+                "name": "ubuntu-16.04*",
```

Comment:
Note to self: Regexp doesn't work here.

Reply:
Checked the OpenStack docs for the Image Service. They match full image names. I will fix that so it's not misleading users who aren't familiar with the Image Service.

Reply:
Tiniest commit ever. It's pushed now!